pygcdl.py

```diff
@@ -346,21 +364,24 @@ def _submit_subset_query(self, query_str, param_dict, dsn, req_name):
                 req_spatial = param_dict["geom_guid"][q]
                 params["geom_guid"] = req_spatial
             r = requests.get(query_str, params=params, headers=headers)
-            print(r.url)
-            if r.status_code >= 400:
+            # print(r.url)
+            if not r.ok:
                 print("Status_code: ", r.status_code)
-                print(r.text)
+                if 'application/json' in r.headers.get('Content-Type'):
+                    raise Exception(r.json()["detail"])
+                else:
+                    raise Exception(r.text)
             else:
                 # Write contents to zip file in chunks
                 with open(subset_zip, 'wb') as f:
                     for chunk in r.iter_content(chunk_size=8192): 
                         f.write(chunk)
                 # Unzip subset_zip
                 with zipfile.ZipFile((str(subset_zip)), 'r') as f:
-                    print("Files downloaded and unzipped: ", f.namelist())
+                    # print("Files downloaded and unzipped: ", f.namelist())

```

Comment:
Should this commented out line be deleted? See line 367 above as well.

Reply:
Fixed on fork